src/components/MoviesList/MoviesList.js

```diff
@@ -1,6 +1,22 @@
 import React from 'react';
 import './MoviesList.scss';
+import PropTypes from 'prop-types';
+import { MovieCard } from '../MovieCard';
 
-export const MoviesList = () => (
-  <>Put the list here</>
+export const MoviesList = ({ movies }) => (
+  <>
+    {movies.map(movie => (
+      <div className="card" key={movie.imdbId}>
+        <MovieCard {...movie} />
+      </div>
```

Comment:
Move this `div` inside `MovieCard` component. Use fragments only when necessary.
```suggestion
      <MovieCard {...movie} key={movie.imdbId} />
```